integrations/web-booking-form.md

```diff
@@ -41,3 +41,8 @@ branchId: `-KmdXp8GIqM0ypiX1lV4`
 Example:
 
 https://inline.app/booking/-KXIv2zV1CA0OJEVKDDm/-LqZC7ERyO2hLDrIik9P?pre_filled_form=ewogICJsYW5ndWFnZSI6ICJ6aC1UVyIsCiAgImN1c3RvbWVyIjogewogICAgIm5hbWUiOiAiaW5saW5lIiwKICAgICJnZW5kZXIiOiAwCiAgfSwKICAidGhpcmRQYXJ0eU1lbWJlciI6IHsKICAgICJpZCI6ICIxMjMiLAogICAgInR5cGUiOiAidmlwIiwKICAgICJwcm92aWRlciI6ICJnYiIsCiAgICAibmFtZSI6ICJpbmxpbmUiLAogICAgImdlbmRlciI6IDAsCiAgICAiYmlydGhkYXkiOiAiMTk4Ni0wNi0wNiIsCiAgICAibGFuZ3VhZ2UiOiAiemgtdHciCiAgfQp9
+
+# Please Note 注意事項
+
+If the outcome from base64 encoding contains `+` characters, please do URL encoding to replace `+` with `%2B` before appending into target URL. The purpose is to ensure your pre_filled_form can be decode correctly. 
```

Comment:
The value should be URL encoded no matter what, not just for `+`. So how about just simply:

> The base64 encoded value for `pre_filled_form` must be URL encoded.

Reply:
How about add that sentence to make URL encoding a general matter. And also keep the `+` case as an example?

Reply:
Sound good to me. So it would be:

> The base64 encoded value for `pre_filled_form` must be URL encoded. For example, if the outcome from base64 encoding contains `+` characters, all `+` must be encoded to `%2B` before appending into target URL. The purpose is to ensure the `pre_filled_form` can be decoded correctly.